flow/cmd/alerts.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v5"
+
+	"github.com/PeerDB-io/peer-flow/generated/protos"
+)
+
+func (h *FlowRequestHandler) GetAlertConfigs(ctx context.Context, req *protos.GetAlertConfigsRequest) (*protos.GetAlertConfigsResponse, error) {
+	rows, err := h.pool.Query(ctx, "select id, service_type, service_config from alerting_config")
+	if err != nil {
+		return nil, err
+	}
+
+	configs, err := pgx.CollectRows(rows, func(row pgx.CollectableRow) (*protos.AlertConfig, error) {
+		config := &protos.AlertConfig{}
+		err := row.Scan(&config.Id, &config.ServiceType, &config.ServiceConfig)
```

Comment:
could give https://pkg.go.dev/github.com/jackc/pgx/v5#RowToStructByName a shot

Reply:
Don't trust it with protos